web-common/src/features/dashboards/time-dimension-details/TDDExportButton.svelte

```diff
@@ -0,0 +1,127 @@
+<script lang="ts">
+  import { WithTogglableFloatingElement } from "@rilldata/web-common/components/floating-element";
+  import { Menu, MenuItem } from "@rilldata/web-common/components/menu";
+  import { getStateManagers } from "@rilldata/web-common/features/dashboards/state-managers/state-managers";
+  import {
+    V1ExportFormat,
+    createQueryServiceExport,
+  } from "@rilldata/web-common/runtime-client";
+  import { onMount } from "svelte";
+  import { get } from "svelte/store";
+  import CaretDownIcon from "../../../components/icons/CaretDownIcon.svelte";
+  import { useDashboard } from "../selectors";
+  import exportTDD from "./export-tdd";
+
+  export let includeScheduledReport: boolean;
+  export let metricViewName: string;
+
+  let exportMenuOpen = false;
+  let showScheduledReportDialog = false;
+
+  const ctx = getStateManagers();
+  const metricsView = useDashboard(get(ctx.runtime).instanceId, metricViewName);
```

Comment:
Probably should avoid using `get` inside svelte components and directly use the `$` accessor.